Whiskey/Tasks/Invoke-WhiskeyNUnit3Task.ps1

```diff
@@ -0,0 +1,310 @@
+
+function Invoke-WhiskeyNUnit3Task
+{
+    <#
+    .SYNOPSIS
+    Runs NUnit3 tests.
+
+    .DESCRIPTION
+    The NUnit3 task runs NUnit tests. The latest version of NUnit 3 is downloaded from nuget.org for you (into a `packages` directory in your build root).
+    
+    The `Path` parameter is mandatory and must contain a list of paths, relative to your `whiskey.yml` file, to the assemblies for which NUnit tests should be run.
+
+    By default, OpenCover and ReportGenerator are also run against the results of the NUnit tests and their output is saved to the `OutputDirectory` of the build. If you wish to **only** run NUnit tests, then specify the `DisableCodeCoverage` parameter with the value of `true`.

```

Comment:
Change to `By default, OpenCover runs NUnit and gathers test code coverage, saving its report to '.output\openCover\openCover.xml'. ReporterGenerator is used to convert the OpenCover report into an HTML report, viewable to '.output\openCover\index.html'. If you wish to...`.

Reply:
Fixed.